shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -81,12 +81,46 @@ - (void)layoutSubviews {
   [super layoutSubviews];
 }
 
+#if FLUTTER_SHELL_ENABLE_METAL
+static bool UseMetalRenderer() {
+  // If there is a command line argument that says Metal should not be used, that takes precedence
+  // over everything else. This allows disabling Metal on a per run basis to check for regressions
+  // on an application that has otherwise opted into Metal on an iOS version that supports it.
+  if ([[[NSProcessInfo processInfo] arguments] containsObject:@"--disable-metal"]) {
+    return false;
+  }
+
+  // If the application want to use metal on a per run basis with disregard for version checks or
+  // plist based opt ins, respect that opinion. This allows selectively testing features on older
+  // version of iOS than those explicitly stated as being supported.
+  if ([[[NSProcessInfo processInfo] arguments] containsObject:@"--force-metal"]) {
+    return true;
+  }
+
+  // This is just a version we picked that is easy to support and has all necessary Metal features.
```

Comment:
May be worth mentioning here that we picked this because of Skia.

Reply:
Does it matter?

Reply:
Someone reading this later might wonder, right?  The constraint for lowering this is on Skia rather than on the engine team directly.